kernel/old/drivers/clock.c

```diff
@@ -1,4 +1,4 @@
-#include <stdint.h>
+#include "../../src/klibc//include/stdint.h"
```

Comment:
Any reason for the "//" or is it just a typo?

Reply:
clion converted some paths automatically

Reply:
it's in the "old" part anyway